bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfile.java

```diff
@@ -218,7 +223,8 @@ public void initFromThingType(String name) {
         isHT = thingType.equals(THING_TYPE_SHELLYHT_STR) || thingType.equals(THING_TYPE_SHELLYPLUSHT_STR);
         isDW = thingType.equals(THING_TYPE_SHELLYDOORWIN_STR) || thingType.equals(THING_TYPE_SHELLYDOORWIN2_STR)
                 || thingType.equals(THING_TYPE_SHELLYBLUDW_STR);
-        isMotion = thingType.startsWith(THING_TYPE_SHELLYMOTION_STR);
+        isMotion = thingType.startsWith(THING_TYPE_SHELLYMOTION_STR)
+                || thingType.equals(THING_TYPE_SHELLYBLUMOTION_STR);
```

Comment:
the thingType.equals(THING_TYPE_SHELLYBLUMOTION_STR); is redundant, every time when equals evaluates to true the startsWith will evaluate to true too. So if startsWith is sufficient the || equals can be omitted.

Reply:
Note, it's THING_TYPE_SHELLYMOTION_STR and THING_TYPE_SHELLY**BLU**MOTION_STR